src/pages/registration/SameNameWarning.tsx

```diff
@@ -0,0 +1,42 @@
+import OpenInNewOutlinedIcon from '@mui/icons-material/OpenInNewOutlined';
+import { Box, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { dataTestId } from '../../utils/dataTestIds';
+import { UrlPathTemplate } from '../../utils/urlPaths';
+
+interface SameNameWarningProps {
+  name: string;
+}
+
+export const SameNameWarning = ({ name }: SameNameWarningProps) => {
+  const { t } = useTranslation();
+  return (
+    <Box
+      sx={{
+        bgcolor: 'secondary.light',
+        width: '100%',
+        padding: '0.75rem',
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '0.5rem',
+      }}>
+      <Box sx={{ bgcolor: 'primary.light', color: 'white', p: '0.5rem', borderRadius: '0.25rem' }}>
+        {t('registration.description.duplicate_title_warning')}
+      </Box>
```

Comment:
I'm all for the new component `<SameNameWarning ... />`, but I was thinking of a `<Notification title="" description="" type="info|warning" />` component just for this blue box. Can't say I have seen a notification like this elsewhere, so it's okay to leave here for now. But if there are more notifications like this in the future we should probably extract it into a component.

Reply:
Good point!